.github/workflows/deploy-prod-updates.yml

```diff
@@ -34,3 +34,20 @@ jobs:
           aws-secret-access-key: ${{ secrets.UPDATES_PROD_AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-2
       - run: aws s3 sync --acl public-read rumba-updates/ s3://updates-prod-developer-mozilla-e2b95ed55d379b1a/rumba-bcd-updates/
```

Comment:
Do we need to invalidate CDN after this step, or doesn't this file get cached?

Reply:
This file does not. But you have reminded me that on successful update from API we do need to invalidate the /api/v2/updates/ cache.

Reply:
Resolved via a16267808705afd93514bc6cdd218bf594ff55d8.